docs/modules/indexes/document_loaders/examples/confluence.ipynb

```diff
@@ -48,6 +49,21 @@
     ")\n",
     "documents = loader.load(space_key=\"SPACE\", include_attachments=True, limit=50)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
can we add a header for this section and a little bit of text describing what is going on here

Reply:
@hwchase17 done. let me know if I need to clarify anything else in the documentation.